src/main/java/main/movie/dao/TicketDao.java

```diff
@@ -0,0 +1,7 @@
+package main.movie.dao;
+
+import main.movie.model.Ticket;
+
+public interface TicketDao {
+    Ticket create(Ticket ticket);
```

Comment:
I would say that it is nice to be consistent in method names
let's rename this to `add()`